orders/models.py

```diff
@@ -71,3 +72,51 @@ def cancel_order(self):
 
     def get_order_total_price(self):
         return self.price_per_day * (self.end_date - self.start_date).days
+
+    def get_order_status(self):
+        if self.status == StatusOptions.Pending:
+            return "Pending"
+        elif self.status == StatusOptions.Approved:
+            return "Approved"
+        elif self.status == StatusOptions.Canceled:
+            return "Canceled"
+        elif self.status == StatusOptions.OnGoing:
+            return "Ongoing"
+        return "Finished"
+
+    @staticmethod
+    def get_capacity_of_daycare_in_dates_range(daycare_id, start_datetime, end_datetime):
```

Comment:
I think it means , it is unclear what kind of type you are expecting to get as "start_datetime" and "end_datetime" for instance you can use ` def get_capacity_of_daycare_in_dates_range(daycare_id: int, start_datetime: DateField, end_datetime: DateField):` etc..

Reply:
As I see it, daycare_id is pretty explicit as an integer, because that's how the member is defined. Same goes for "start_datetime" and "end_datetime".
Tell me if what do you think.

Reply:
Added typing.